### PR TITLE
Improved Classification Documentation

### PR DESCRIPTION
**Pull request by ManishAradwad**
Fixes: https://github.com/tensorflow/tensorflow/issues/34631

### CODE REVIEW

File: site/en/tutorials/keras/classification.ipynb [hunk 1]
```diff
@@ -637,7 +663,9 @@
         "id": "yWfgsmVXCaXG"
       },
       "source": [
-        "It turns out that the accuracy on the test dataset is a little less than the accuracy on the training dataset. This gap between training accuracy and test accuracy represents *overfitting*. Overfitting is when a machine learning model performs worse on new, previously unseen inputs than on the training data."
+        "It turns out that the accuracy on the test dataset is a little less than the accuracy on the training dataset. This gap between training accuracy and test accuracy represents *overfitting*. Overfitting is when a machine learning model performs worse on new, previously unseen inputs than on the training data. When the trained model 'memorises' the training data, it's accuracy on test data reduces as compared with training data and model is said to be overfitting the training data. For more information, see below:\n",
+        "*   [Demonstration of Overfitting](https://www.tensorflow.org/tutorials/keras/overfit_and_underfit#demonstrate_overfitting)\n",
```
Comment (lamberta) [inline site/en/tutorials/keras/classification.ipynb]: Please use relative links so these also work in GitHub.
For spaces in anchor links, use '-' instead of '_'

File: site/en/tutorials/keras/classification.ipynb [hunk 2]
```diff
@@ -637,7 +663,9 @@
...
+        "*   [Strategies to prevent Overfitting](https://www.tensorflow.org/tutorials/keras/overfit_and_underfit#strategies_to_prevent_overfitting)"
```
Comment (lamberta) [inline site/en/tutorials/keras/classification.ipynb]: Same as above

File: site/en/tutorials/keras/classification.ipynb [hunk 3]
```diff
@@ -637,7 +663,9 @@
         "id": "yWfgsmVXCaXG"
       },
       "source": [
-        "It turns out that the accuracy on the test dataset is a little less than the accuracy on the training dataset. This gap between training accuracy and test accuracy represents *overfitting*. Overfitting is when a machine learning model performs worse on new, previously unseen inputs than on the training data."
+        "It turns out that the accuracy on the test dataset is a little less than the accuracy on the training dataset. This gap between training accuracy and test accuracy represents *overfitting*. Overfitting is when a machine learning model performs worse on new, previously unseen inputs than on the training data. When the trained model 'memorises' the training data, it's accuracy on test data reduces as compared with training data and model is said to be overfitting the training data. For more information, see below:\n",
```
Comment (lamberta) [inline site/en/tutorials/keras/classification.ipynb]: > When the trained model 'memorises' the training data, it's accuracy on test data reduces as compared with training data and model is said to be overfitting the training data.

This sentence is a bit difficult to parse. Can you please shorten/clarify?
Also, use en-US spelling ("memoize") and "it's" = "its"

File: site/en/tutorials/keras/classification.ipynb (hunk 1 above)
Comment (ManishAradwad) [reply site/en/tutorials/keras/classification.ipynb]: I'm sorry @lamberta but what do u mean by relative links. Also, when I changed '_' to '-' the links show 404 error

https://www.tensorflow.org/tutorials/keras/overfit-and-underfit#strategies-to-prevent-overfitting
https://www.tensorflow.org/tutorials/keras/overfit-and-underfit#strategies-to-prevent-overfitting

Comment (lamberta) [reply site/en/tutorials/keras/classification.ipynb]: Like this: [strategies to prevent overfitting](./overfit-and-underfit.ipynb#strategies-to-prevent-overfitting)

This should work when browsing GitHub and will also work on tensorflow.org when we generate the html.

Comment (ManishAradwad) [reply site/en/tutorials/keras/classification.ipynb]: Do u mean like this??
[Demonstrate Overfitting](https://github.com/tensorflow/docs/blob/master/site/en/tutorials/keras/overfit_and_underfit.ipynb#demonstrate-overfitting)
